Android/app/build.gradle

```diff
@@ -54,26 +51,39 @@ android {
 
     buildFeatures {
         viewBinding true
+        compose true
     }
+    namespace 'live.ditto.inventory'
 }
 
 dependencies {
-    implementation("live.ditto:ditto:4.7.2")
+    implementation("live.ditto:ditto:4.9.3")
 
-    // Load Presence Viewer from an .aar library file
-    implementation fileTree(dir: "${rootDir}", include: ["*.aar"])
+    // Ditto Tools
+    implementation "live.ditto:ditto-tools-android:4.0.1"
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'androidx.appcompat:appcompat:1.6.1'
-    implementation 'androidx.fragment:fragment-ktx:1.5.7'
-    implementation 'androidx.core:core-ktx:1.9.0'
-    implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+    implementation 'androidx.appcompat:appcompat:1.7.0'
+    implementation 'androidx.fragment:fragment-ktx:1.8.5'
+    implementation 'androidx.core:core-ktx:1.15.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.2.0'
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
-    implementation 'androidx.recyclerview:recyclerview:1.3.0'
+    implementation 'androidx.recyclerview:recyclerview:1.4.0'
+
+    // Import the Compose BOM
+    implementation(platform("androidx.compose:compose-bom:2025.01.00"))
+
+    // Compose
+    implementation("androidx.compose.ui:ui")
+    implementation("androidx.compose.material3:material3")
+    implementation("androidx.compose.ui:ui-tooling-preview")
+    debugImplementation("androidx.compose.ui:ui-tooling")
+    debugImplementation("androidx.compose.ui:ui-test-manifest")
+    implementation("androidx.activity:activity-compose")
 
     testImplementation 'junit:junit:4.13.2'
-    androidTestImplementation 'androidx.test.ext:junit:1.1.5'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
+    androidTestImplementation 'androidx.test.ext:junit:1.2.1'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.6.1'
```

Comment:
Just a comment so feel free to ignore - but should be keep updating the dependencies of test frameworks at all if the sample project doesn't actually contain any tests? 
Same for `test` and `androidTest` source sets